test_cli_backend_basic.py

```
@@ -0,0 +1,145 @@
+    print("Testing Agent with CLI backend...")
+    try:
+        from praisonaiagents import Agent
+        
+        # Test with string backend ID
+        agent = Agent(
+            name="test_agent",
+            role="Test Assistant", 
+            cli_backend="claude-code"
+        )
+        
+        print(f"✓ Agent created with CLI backend")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)